css/main.css

```diff
@@ -322,6 +341,33 @@ li {
   outline: none;
 }
 
+#mobile-nav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  z-index: 10;
+  background-color: #3c3a39;
+}
```

Comment:
- [ ] [OPTIONAL] When you first open the application, the menu is already expanded this is due to the fact that it is not hidden initially. in order to make your application give a good first impression, it might be a good idea to hide the menu when the user opens it for the first time.